### PR TITLE
Feat: high_availability - role creation

### PR DESCRIPTION
**Pull request by oxedions**
HA role based on Corosync-Pacemaker

Handle cluster creation and resources

### CODE REVIEW

**Review by ginomcevoy — CHANGES_REQUESTED** [review]
Looks pretty good overall. just added the following comments:
- Typo in "Set properties" task
- Added one last task to disable stonith if there is no stonith configuration
- Added a recommendation on how to execute a HA playbook with this role in the README

I would say that only the first change is really necessary.

File: roles/high_availability/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+  package:
+    name: "{{item}}"
+    state: present
+  tags:
+    - package
+  loop:
+    - pacemaker
+    - corosync
+    - pcs
+
```
Comment (ginomcevoy) [inline roles/high_availability/tasks/main.yml:12]: Add this RPM:
```suggestion
    - fence-agents-all
    
```

**Comment by oxedions** [comment]
Dear @ginomcevoy 

Thanks a lot for this feedback ! I will integrate all of this 🙂

File: roles/high_availability/tasks/main.yml [hunk 2]
```diff
@@ -105,7 +105,7 @@
 
     - name: command █ Set properties
       command: "pcs property set {{ item.name }}={{ item.value }}"
-      loop: "{{phigh_availability_pcs_propert | default([]) }}"
+      loop: "{{high_availability_pcs_propert | default([]) }}"
```
Comment (ginomcevoy) [inline roles/high_availability/tasks/main.yml:108]: Still tiny typo:
```
      loop: "{{high_availability_pcs_property | default([]) }}"
```

Comment (oxedions) [reply roles/high_availability/tasks/main.yml:108]: I forgot to save the file 😄 

Thanks a lot for checking that one. Let me commit.

Comment (oxedions) [reply roles/high_availability/tasks/main.yml]: Should be ok now

**Pull request merged**